shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -51,6 +51,17 @@
           <button class="close-button" title="Close {{title}}"></button>
         </li>
         {{/each}}
+
+        {{#with accountExpires}}
+          <li><div class="demo-notice">
+            <p>This demo expires in: <span class="countdown {{#if urgent}}urgent{{/if}}">{{countdown}}</span></p>
+
+            <p>To make your data permanent, create an account:
+              <button class="sign-in">Create Account »</button></p>
```

Comment:
Actually, standard style within Sandstorm is to capitalize the first word of the action and no others - see our other "Sign in", "Share access", "Send an invite", "Get shareable link", and so on.